### DISCUSSION TITLE
Hey.

### DISCUSSION THREAD

**Discussion by TesteTheTestCat** [DesModder/DesModder#689] [Q&A] [answered]
My graph isnt showing up.
![image](https://github.com/DesModder/DesModder/assets/140556085/413e3751-ffee-4527-bbfc-b69a0028167a)

**Comment by radian628** [comment]
Thanks for the bug report! What browser are you using on what OS? I can't seem to replicate this in Edge or Firefox on Windows 11. Also, if possible, can you send a link to the graph?

**Reply by TesteTheTestCat** [reply]
I'm using Google Chrome on Windows 11, and the graph is literally just x²

**Reply by radian628** [reply]
Just tested this on Chrome (latest version) on Windows 11 with the exact same expression (x²). It seems to work just fine for me. Can you send me a link to the graph? I have a hunch that something in the graph state might be wrong, making the graph *appear* like a regular x² but actually containing some extra (or missing) data that prevents it from displaying. If you send me a graph link (if you don't have a Desmos account, you can do this with this popup here), I can check to see if anything's off about it: 
![image](https://github.com/DesModder/DesModder/assets/70352880/aa2458b3-4b94-4986-88e8-5f258ee64e07)

If that doesn't work, my next best guess is that you might need to update your browser or restart your computer. I know, very generic advice lol, but sometimes it just works for some reason.

**Reply by TesteTheTestCat** [reply]
it is just x², nothing new, just x², also, my browser is fully updated and, incase this is a weirder issue, i have 8 gigabytes of ram (if that helps)

**Reply by radian628** [reply]
Try: 
1. loading the page, 
2. pressing `F12` (or `Ctrl+Shift+I` if that doesn't work) to open DevTools, 
3. going to the "console" tab
4. and taking a screenshot with all the messages in the console, like the one below 
![image](https://github.com/DesModder/DesModder/assets/70352880/bf64cb6b-a2ed-40af-995d-39270d763e5e)

This should display any errors that might be breaking Desmos or DesModder.

**Reply by radian628** [reply]
Barring that, I'd try uninstalling and reinstalling the extension on the offchance that it's somehow an issue with the way you've configured it.

**Reply by TesteTheTestCat** [reply]
![image](https://github.com/DesModder/DesModder/assets/140556085/014c530f-ac93-4c85-b8fe-b834ddb795d5)
uhhhhhhhh

**Reply by radian628** [reply]
Okay I think I know what's causing this: Your browser/computer don't support WebGL2, which is the browser feature required for running GLesmos, which I assumed you've enabled to get faster implicit inequalities. If you disable the plugin, that _should_ in theory fix the bug. If it doesn't, then we have bigger problems lol.

On a related note, how old is your computer and what graphics card does it have? This will help us figure out whether this is actually the root cause of the problem.

Either way, I should probably implement some kind of warning that GLesmos won't work if someone's browser doesn't support WebGL2.

**Reply by TesteTheTestCat** [reply]
![image](https://github.com/DesModder/DesModder/assets/140556085/235474e7-3a83-42a3-ab6e-caad041a13e0)
uhhhhhh

**Reply by radian628** [reply]
Wait, that was a lot wordier than I needed it to be lol. What I meant to say is that you should disable the GLesmos plugin. That should fix the error hopefully.

**Reply by radian628** [reply]
Okay, something here is definitely wrong lol

**Reply by TesteTheTestCat** [reply]
ill send over my pc specs

**Reply by TesteTheTestCat** [reply]
![image](https://github.com/DesModder/DesModder/assets/140556085/495ca632-89c2-4c1e-8ddc-af5b25008d33)

**Reply by radian628** [reply, marked as answer]
Thanks for the info! i5-2400 is pretty old (assuming integrated graphics here), so WebGL2 support is likely the issue here. I've opened #690 to fix this--- should be a quick fix.

**Reply by TesteTheTestCat** [reply]
alright :3

**Reply by TesteTheTestCat** [reply]
also nice #690

**Reply by radian628** [reply]
nice

**Reply by radian628** [reply]
Oh yeah I forgot to mention it might be worth reloading the page now that you've disabled the GLesmos plugin. Not sure if you already tried that, but if you didn't, it *may* have still been partly loaded in despite disabling it (which is still a bug in and of itself, but at least it isn't one that makes it impossible to use DesModder). Refreshing should get rid of any traces of it.
